### DISCUSSION TITLE
Centering widgets + text slightly transparent for some reason

### DISCUSSION THREAD

**Discussion by ItsVoltz** [AllenDang/giu#142] [Q&A]
Thanks for everyone who has constantly helped me anyways I had a question on how to align all the widgets to the center of my frame. 

I also have an odd bug where text is slightly transparent seen here (the text is supposed to be white yet has a bluish tint from the blue background):
![e943bddf-9c15-45e2-8f7b-c4876f8165b7](https://user-images.githubusercontent.com/29076540/107569250-18400200-6bb6-11eb-8bb9-b02dbda32e90.png)

Any ideas how I can perfectly center my widgets horizontally and fix the weird text transparency (in the styles text is fully opaque)

**Comment by gucio321** [comment]
@ItsVoltz yo've posted some time ago and iirc u've opened an issue later so I hope you've fixed alignment stuff (if not just ping me here)
in case of text color... It seems strange, have you tried enforcing text color by giu.Style ?
maybe the default style has some alpha parameter set for text color...

**Reply by gucio321** [reply]
@ItsVoltz any updates here?
